### DISCUSSION TITLE
Port to Avalonia 11.x

### DISCUSSION THREAD

**Discussion by tomlm** [Consolonia/Consolonia#115] [General]
I <3 Consolonia and would love to see this ported to Avalonia 11.x. 

I would like to contribute to this, but 
* am not certain if there is an ongoing effort to do his
* am not certain exactly the best place to contribute to not duplicate/stomp on other peoples work.

How can I help?

**Comment by jinek** [comment]
@tomlm Welcome!

I've tried to upgrade, had investigated issues and it seems there are no big obstacles. Still, text rendering slightly changed (which is hardly customized in Consolonia) and several abstractions too.

From issues I've seen there I would say you are safe to contribute this part, it does not interfere with any of open tickets.

As a suggestion, may be begin to do the upgrade and push the branch, so I could help. And please feel free to discuss/ask here, both code and idea are still under considerations. Plus, if you need I will be able to guide you with details.

Important thing to note is that RELEASE and DEBUG configurations in the solution give different errors. For development you can just use DEBUG.

**Reply by jinek** [reply]
@tomlm I can also include you to contributors so you don't need to fork it.
